src/components/AppUpdate/AppUpdate.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
+import cx from 'clsx'
+import _isFunction from 'lodash/isFunction'
+import { isElectronApp } from '../../redux/config'
+
+import './style.css'
+
+// @TODO: i18n
+function AppUpdate() {
+  const [hideNotification, setHideNotification] = useState(true)
+  const [hideRestart, setHideRestart] = useState(true)
+  const [message, setMessage] = useState('')
+
+  // init state
+  useEffect(() => {
```

Comment:
Why is it required to re-initialise the state in useEffect here?

Reply:
good catch!, removed now

earlier I added because of habit to reinit state on prop change, but here initial state is not based on prop so can be removed (https://thewebdev.info/2021/03/14/how-to-fix-the-react-usestate-hook-not-setting-initial-value-problem/)